src/en/help-oracle.md

```diff
@@ -0,0 +1,180 @@
+
+### Steps to associate images:
+
+Sign in to [https://cloud.oracle.com/home][cloudoracle] with your credentials.
+You will need to know your Data Center and Identity Domain.
+
+1. If you're not on the *Dashboard*, click the `Dashboard` button.
+1. From the Dashboard, click on the green `Create Instance` button.
+1. Select `Compute` from the pop-up windows; you'll come to the Compute screen.
+1. Select `Marketplace` from the list on the left of your screen.
+1. Search for Ubuntu and click `Select` on an image matching one of those listed above.
+1. Accept the terms of use in the 'Install Marketplace Image' pop-up and click `Install`.
+1. Informational messages on install status will appear in the upper right corner.
+1. You can cancel the rest of the image creation at this point.
+
+To verify, go back to the Dashboard. Click on the `Compute` link (one
+of the boxes on your Dashboard). From the 'Service Details: Oracle Compute Cloud
+Service' window, click on `Open Service Console` and select the `Images` tab.
+You'll see a drop down menu; change `Category: All` to `Category: Personal` and
+you should see your image. 
+
+## Set up the Oracle cloud
+
+If using a paid Oracle account, there's one currently supported region,
+`uscom-central-1`, which is supported out of the box.
+
+Using a trial account requires adding the cloud information to Juju using the
+`add-cloud` command:
+
+```bash
+juju add-cloud
+```
+
+From the list of clouds, enter `oracle-compute`:
+
+```no-highlight
+Cloud Types
+  maas
+  manual
+  openstack
+  oracle-compute
+  vsphere
+
+Select cloud type: oracle-compute
+```
+You will then be asked for a name and an endpoint before being informed that
+the cloud has been successfully added:
+
+```no-highlight
+Enter a name for your oracle-compute cloud: myoracle
+Enter the API endpoint url for the cloud: https://compute.uscom-central-1.oraclecloud.com/
+Cloud "myoracle" successfully added
+```
+The endpoint can be found in the `REST Endpoint` field in the Dashboard's
+`Compute` screen. The above steps also work for a paid account and are useful
+if there's a need to define the default model configuration or other bespoke
+bootstrap configuration.
+
+## Set up credentials
+
+To access the cloud, you first need to add credentials using the cloud name
+configured previously: 
+
+```bash
+juju add-credential myoracle
+```
+
+You will be asked for your username, password and Identity Domain:
+
+```no-highlight
+Enter credential name: default
+Using auth-type "userpass".
+Enter username: user@somewhere.com
+Enter password: 
+Enter identity-domain: mydomain
+Credentials added for cloud myoracle.
+```
+
+## Start using Juju
+
+You can now start using Juju with your Oracle cloud:
+
+```bash
+juju bootstrap myoracle
+juju deploy ...
+```
+
+## Storage
+
+Juju includes support for Oracle block storage volumes. The storage pool name
+to use is `oracle`:
+
+```bash
+juju deploy postgresql --storage data=oracle,10G
+```
```

Comment:
This might throw inexperienced users. Consider mentioning that block storage is optional.